earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +47,21 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def working_dir = "../src/"
+
+// Check for either "-PRELEASE" or "-POPTIMIZE" command line parameters.
+// Return the correct directory based on which paramater is used.
+def native_x86 = {
+  if ( project.hasProperty( 'OPTIMIZE' ) ) { return "NATIVE-OPT-x86_64" }
```

Comment:
Minor: We don't have an issue ticket that specifies what build targets should be supported, but what `scons` supports is:

```Python
if release:
  builddir = 'REL-'
elif optimize:
  builddir = 'OPT-'
elif gprof:
  builddir = 'PROF-'
else:
  builddir = 'DBG-'
if sandbox:
  builddir += 'SBOX-'
builddir += march

if native_cc:
  builddir = 'NATIVE-' + builddir
```

Currently, this code partially supports `release=1`, `optimize=1` and `debug=1`, but not `profile=1`, or `sandbox=1`.  (I'm not sure anyone has tested cross-compiling.)

Should we support what `scons` supports, or write another ticket to refactor the two to be more consistent?

Reply:
Minor: The project variable, and the variables passed to `scons` are not in all capitals.  I think it would easier, if users didn't have to remember when to capitalize, and when not to.